task1/src/csv.js

```diff
@@ -0,0 +1,48 @@
+import { appendFileSync, existsSync, unlinkSync } from 'fs';
+import { csv } from 'csvtojson';
+
+const csvFilePath = './csv/input.csv';
+const txtFilePath = './txt/output.txt';
+
+deleteTxtFile(txtFilePath);
+convertCsvData();
+
+function convertCsvData() {
+  csv()
+    .fromFile(csvFilePath)
+    .subscribe(
+      (jsonData) => writeJsonToTxt(jsonData),
+      (err) => console.error('Error:', err),
+      () => console.log('Success')
+    );
+};
+
+function writeJsonToTxt(jsonData) {
+  appendFileSync(txtFilePath, prepareJson(jsonData), function (err) {
+    if (err) {
+      console.log(err);
+    }
+  });
+};
+
+function prepareJson(jsonData) {
+  delete jsonData.Amount;
+  jsonData.Price = Number(jsonData.Price);
+  Object.keys(jsonData).forEach(key => {
+    let value = jsonData[key];
+    delete jsonData[key];
+    jsonData[key.toLowerCase()] = value;
+  });
```

Comment:
[Recommendation] Could be done by using Object.entries, excluding Amount, formatting Price,  lower-casing all keys and then Object.fromEntries.

Reply:
the recommendation was applied, thank you